source/wp-content/themes/wporg-parent-2021/templates/single.html

```diff
@@ -9,6 +9,52 @@
 		<!-- wp:post-content {"className":"is-style-two-column-display","layout":{"inherit":true}} /-->
 	</section>
 	<!-- /wp:group -->
+
+	<!-- wp:group {"align":"full","style":{"spacing":{"margin":{"top":"var:preset|spacing|60"}}}} -->
+	<div class="wp-block-group alignfull is-style-two-column-display" style="margin-top:var(--wp--preset--spacing--60)">
```

Comment:
If there are no comments, will this still render the margin?

Reply:
Yeah, it does, so there's a bigger gap here when there are no comments. I've moved the margin to the `comments` block and now it only shows up when there are comments or the comment form is open — https://github.com/WordPress/wporg-parent-2021/pull/104/commits/c97b7012a499ed0313f9391e5a7a3331ce311f31